src/Accessibility.ts

```diff
@@ -108,6 +110,32 @@ export interface Accessibility {
    * `null` indicates there is no media, `undefined` or missing property indicates unknown.
    */
   media?: ArrayLike<Media> | null;
+  /**
+   * `true` if the facility provides mobility equipment e.g. foldable wheelchairs or crutches
+   */
+  providesMobilityEquipment?: boolean;
+  /**
+   * Describes the pick up or drop off zone
+   */
+  pickUpOrDropOffZone?: Ground | null;
+  /**
+   * `true if the room is on an accessible path or on the accessible route`
+   * `false` if not, `undefined` if the condition is unknown or difficult to assess.
+   */
+  hasWheelchairAccessiblePathFromOutside?: boolean;
+  /**
+   * Describes the doors of the place
```

Comment:
```suggestion
   * Describes the place’s doors.
```

Reply:
Related to `’` just a minor hint: you used `'` later on, which i would rather use instead of `’`.

Reply:
Then these places should be changed, too!

> When text is set, U+2019 right single quotation mark is preferred as apostrophe, but only U+0027 is present on most keyboards.

([Unicode 9.0.0 standard, ch. 6, p. 276](https://www.unicode.org/versions/Unicode9.0.0/ch06.pdf))